### PR TITLE
Add windows support

### PR DESCRIPTION
**Pull request by j3ddesign**
Currently this plugin doesn't work on windows, due to it's need to be different.

This may not be the best way to implement it, but it's working for me so thought I'd submit :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+        });
+      }
+    });
+  }
+}
 
 exports.middleware = (store) => (next) => (action) => {
   switch (action.type) {
+    case 'SESSION_ADD_DATA':
+      if (process.platform === 'win32' && dirRegex.test(action.data)) curDir = dirRegex.exec(action.data)[1];
```
Comment (hharnisc) [inline index.js]: seems like you could call setCwd with the current directory here (`dirRegex.exec(action.data)[1]`).

File: index.js [hunk 2]
```diff
@@ -1,23 +1,39 @@
...
+const dirRegex = /([\w]:[\s\S]+)>/;
 
-const setCwd = (pid) =>
-  exec(`lsof -p ${pid} | grep cwd | tr -s ' ' | cut -d ' ' -f9-`, (err, cwd) => {
-    if (err) {
-      console.error(err);
-    } else {
-      cwd = cwd.trim();
+const setCwd = (pid) => {
+  if (process.platform === 'win32') {
```
Comment (hharnisc) [inline index.js]: If you made the change described below, I think you might not need this anymore.

**Comment by j3ddesign** [comment]
I won't be able to get to this within the next few weeks as I have a pretty major launch date I'm working towards. Once that's over, if It's not been solved by then, I'll be happy to update the PR :)

**Comment by colbin8r** [comment]
#17

**Comment by hharnisc** [comment]
Fixed this one in #25 - thank you for the PR @j3ddesign

**Pull request closed without merging**